src/shared/svelte-compile-warns/transform/typescript.ts

```diff
@@ -29,7 +29,7 @@ export function transform(
     const output = ts.transpileModule(code, {
       reportDiagnostics: false,
       compilerOptions: {
-        target: ts.ScriptTarget.ESNext,
+        target: context.parserServices.program?.getCompilerOptions()?.target || ts.ScriptTarget.ESNext,
```

Comment:
It doesn't seem to work for our test cases. I think we probably need to add an option like this:

```suggestion
        target: context.parserServices.program?.getCompilerOptions()?.target || ts.ScriptTarget.ESNext,
        module: ts.ModuleKind.ESNext,
```

Also, Could you format it using `yarn lint-fix`?